Assets/Tests/InputSystem/CorePerformanceTests.cs

```diff
@@ -123,8 +123,13 @@ public void Performance_ReadEveryKey()
 
         Measure.Method(() =>
         {
+            int keyIndex = 0;
             foreach (var key in keyboard.allKeys)
+            {
+                if (++keyIndex == (int)Key.IMESelected)  // Skip IMESelected as it's not a real key.
```

Comment:
Why this conditional check in a perf test? I am afraid it may color the perf mark more than adding some usefulness to the test itself?

Reply:
I see, thanks for pointing this out.